cmd/import.go

```diff
@@ -42,101 +47,49 @@ func runImport(cmd *cobra.Command, args []string) {
 			// fmt.Printf("Found Existing Service: %s, %s\n", foundService.Name, foundService.Id)
 			continue
 		}
-		newService, err := client.CreateService(opslevel.ServiceCreateInput{
+		serviceCreateInput := opslevel.ServiceCreateInput{
 			Name:        service.Name,
 			Product:     service.Product,
 			Description: service.Description,
 			Languague:   service.Language,
 			Framework:   service.Framework,
-			// TODO: Tier
 			// TODO: Owner
-			// TODO: Lifecycle
-		})
+		}
+		if v, ok := tiers[service.Tier]; ok {
```

Comment:
I'm not too jazzed about this as it means validation doesn't happen until import time and it fails silently by not assigning the Tier or Lifecycle

but the alternative means that `service preview` will require an OpsLevel API Token - where as right now it does not.

I'm not sure how impactful this is - as once we get reconciliation it no longer an issue really - we would just need to log that we are skipping the Tier or Lifecycle because it doesn't match an existing alias